examples/decoding/plot_receptive_field.py

```diff
@@ -0,0 +1,122 @@
+"""
+=========================================
+Receptive Field Estimation and Prediction
+=========================================
+
+This example re-creates figures from a paper from the Lalor group which
+demos the mTRF toolbox in Matlab _[1]. We will show how the
+:class:`mne.decoding.ReceptiveField` class can perform a similar function
```

Comment:
you still need to add an entry to `python_reference.rst` for this and `delay_time_series`, link is currently broken